martian/src/utils.rs

```diff
@@ -52,12 +52,14 @@ pub fn to_camel_case(stage_name: &str) -> String {
 /// Parse the `env::args()` and return the name of the
 /// current executable as a String
 pub fn current_executable() -> String {
-    let args: Vec<_> = std::env::args().collect();
-    std::path::Path::new(&args[0])
+    Path::new(&std::env::args().next().unwrap())
+        .canonicalize()
```

Comment:
Please revert this.  Cannonicalization makes this function essentially useless in a bazel runfiles tree.